cg/apps/demultiplex/sample_sheet/novaseq_sample_sheet.py

```diff
@@ -88,21 +82,34 @@ def convert_sample_to_header_dict(
         sample_dict = sample.dict(by_alias=True)
         return [str(sample_dict[header]) for header in sample_sheet_headers]
 
+    def get_additional_sections_sample_sheet(self) -> List[List[str]]:
+        """Build all sections of the sample sheet that is not the Data section."""
+        return [
+            [SampleSheetNovaSeq6000Sections.Settings.HEADER.value],
```

Comment:
Don't think `.value` is necessary here 😄

Reply:
> > @seallard I will try testing without adding `.value` to see if the sample sheet gets the correct output, bt at least in the case of `SampleSheetNovaSeq6000Sections.Data.COLUMN_NAMES` it fails without adding `.value`
> 
> Ah, maybe it does not work for that field! I think it should be ok for
> 
> * `SampleSheetNovaSeq6000Sections.Data.FLOW_CELL_ID`
> * `SampleSheetNovaSeq6000Sections.Data.HEADER`
> * `SampleSheetNovaSeq6000Sections.Settings.HEADER`
> 
> And maybe some others that I might have missed, not 100 % sure though

It fails if any line of the sample sheet has it, see here:
```
>           if len(line) <= 5:
E           TypeError: object of type 'Settings' has no len()
```